### ISSUE TITLE
The 'Pods_XXX_Example' target has transitive dependencies that include static binaries: (/MySwiftApp/Pods/QuickBlox/Quickblox.framework

### ISSUE THREAD

**Issue by subrajits** [QuickBlox/quickblox-ios-sdk#557]
Hi,

I am using Swift  and trying to install

      
```
use_frameworks!
        pod ‘Bolts’
	pod 'QMChatViewController'
	pod 'QMChatViewController'
	pod 'QuickBlox'
	pod 'QMServices'
	pod 'SDWebImage', '~>3.8'
	pod 'SVProgressHUD'
	pod 'SearchEmojiOnString', '~> 1.0'
```

Getting this error for "QuickBlox/Quickblox" 
Please let me know how can i resolve it, Any Help will be Appreciated. 

Thanks,

**Comment by Raikerian** [comment]
Hi,

Our current Quickblox framework is built as static library and do not support use_frameworks! flag.
At this moment we are working on a new dynamic version of framework and will release it approximately next week (iOS 8+ for dynamic version). Right now the possible solution for you is to remove use_frameworks! flag, or install Quickblox framework, QMChatViewController and QMServices [manually](http://quickblox.com/developers/IOS-how-to-connect-Quickblox-framework#Framework) (without cocoapods).

**Comment by aanchalkalani** [comment]
Hi,

If we try to add Quickblox framework Manually (without cocoapods) as mention in the above link, **i got 114 errors**, like (it's an example) :- `QuickbloxWebRTC(webrtc_h264_video_toolbox.h264_video_toolbox_encoder.o)
  "_kVTDecompressionPropertyKey_RealTime", referenced from:
      webrtc::H264VideoToolboxDecoder::ResetDecompressionSession() in QuickbloxWebRTC(webrtc_h264_video_toolbox.h264_video_toolbox_decoder.o)
      webrtc::H264VideoToolboxDecoder::ConfigureDecompressionSession() in QuickbloxWebRTC(webrtc_h264_video_toolbox.h264_video_toolbox_decoder.o)
  "_kVTEncodeFrameOptionKey_ForceKeyFrame", referenced from:
      webrtc::H264VideoToolboxEncoder::Encode(webrtc::VideoFrame const&, webrtc::CodecSpecificInfo const*, std::__1::vector<webrtc::FrameType, std::__1::allocator<webrtc::FrameType> > const*) in QuickbloxWebRTC(webrtc_h264_video_toolbox.h264_video_toolbox_encoder.o)
  "_kVTProfileLevel_H264_Baseline_AutoLevel", referenced from:
      webrtc::H264VideoToolboxEncoder::ConfigureCompressionSession() in QuickbloxWebRTC(webrtc_h264_video_toolbox.h264_video_toolbox_encoder.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)`


I am using Swift, Xcode 7.3.1
Please let us know how can i resolve it, Any Help will be appreciated.

Thanks,

**Comment by VitaliyGuru** [comment]
@aanchalkalani Are you sure that you're trying to add QuickBlox SDK?
The errors above related to QuickbloxWebRTC.

**Comment by Pro100Andrey** [comment]
Hi @aanchalkalani please include `VideoToolbox.framework` in you project
<img width="682" alt="2016-08-02 11 51 20" src="https://cloud.githubusercontent.com/assets/1109916/17322729/91b2bc26-58a7-11e6-8106-41a23ccf2283.png">

**Comment by aanchalkalani** [comment]
@VitaliyGuru  Yes, i am adding the QuickBlox SDK as mention in the above link [manually](http://quickblox.com/developers/IOS-how-to-connect-Quickblox-framework#Framework)
@Pro100Andrey  I tried that also but still getting the same error.

**Comment by Pro100Andrey** [comment]
@aanchalkalani, please remove QuickbloxWebRTC.framework if you don't use Audio/Video call functionality, and try again

**Comment by aanchalkalani** [comment]
Hi @Pro100Andrey thanks for reply, I removed the `QuickbloxWebRTC.framework` and include VideoToolbox.framework the above error is resolved

**Comment by Raikerian** [comment]
@subrajits we have recently released Quickblox iOS SDK as Dynamic framework. From version 2.7.5 you can use **use_frameworks!** flag.

**Comment by adil-appifytech** [comment]
Any plan to do the same for WebRTC library?

**Comment by Raikerian** [comment]
@adil-appifytech not in a near future. Just install it manually.

**Comment by softninjablade** [comment]
hi @Raikerian , I integrated QuickBlox sdk successfully via cocoapods, but when I submit the app for iOS 10 via Xcode 8 release version, I am getting strange error.

Xcode realise my app id as bundle id of QuickBlox framework and complains its never registered one.
Let me know why this is happening.

I tried with Xcode 7 too, but its same.
Earlier, I have published a lot of apps but this is first time for me.

I couldn't avoid code sign for frameworks when I export the archive file in Xcode.
I look forward your quick response.
Thanks
